torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -196,14 +213,16 @@ def speculate_subgraph(
     source_target=None,
     always_restore=False,
     enable_grad=None,
-    # NOTE [Temporary argument `manually_set_subgraph_inputs`]
-    # If manually_set_subgraph_inputs=True, then we manually add
-    # the `sub_args` to `subgraph`, if False then we rely
-    # on tracer's lifting mechanism to lift these args.
+    # NOTE [Temporary argument `set_subgraph_inputs`]
+    # If set_subgraph_inputs="manual", we manually add the `sub_args` to `subgraph`,
+    # If "automatic", we rely on tracer's lifting mechanism to lift these args.
+    # If "flatten_manual", we first flatten the sub_args into a flattend list, then manually
+    # set the flattend inputs to the subgraph. This saves us from having to mapping the automatically
+    # lifted inputs back to their original position, where ordering of input matters e.g. map.
     # NOTE: Default `True` is temporary and plan is
     #       to always lift args in future and remove this
     #       argument.
-    manually_set_subgraph_inputs=True,
+    set_subgraph_inputs="manual",
```

Comment:
Actually, do we need "manual" anymore? "flatten_manual" seems like it is better than "manual"

Reply:
Nvm, we still need manual if preserving the calling convention matters to people.